### PR TITLE
Amit/update to fc28

### PR DESCRIPTION
**Pull request by amittar-lb**
Not sure if we want to upgrade the entire lab but still better to have an open pr.
No need for merge

### CODE REVIEW

File: inaugurator/main.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+
+# Need to make sure osmosis is installed
+import pkgutil
+if pkgutil.find_loader('osmosis') is None:
```
Comment (yant-lb) [inline inaugurator/main.py:4]: nice!

**Review by yant-lb — APPROVED** [review]
LGTM
do we plan to deploy it?
maybe only on 1 allocator first?

**Comment by amittar-lb** [comment]
> LGTM do we plan to deploy it? maybe only on 1 allocator first?

I was thinking to deploy it in full scale on working hours. if errors will pop then we can always switch it back and `hard_reclaim` the infected servers

**Comment by amittar-lb** [comment]
Maybe I will merge it after we test it in the lab

**Pull request merged**